pkg/app/app.go

```diff
@@ -164,6 +165,29 @@ func (a *App) Start() {
 		a.log.Notice("Broker successfully bootstrapped on startup")
 	}
 
+	interval := a.config.Broker.RefreshInterval
+	if interval != 0 {
+		ticker := time.NewTicker(time.Duration(interval) * time.Second)
```

Comment:
yeah that is correct, my fault, you don't have to cast if you are hard coding 600. 

I actually have an idea and let me know if you hate it but we could offer a lot more flexibility by using [parseDuration](https://golang.org/pkg/time/#ParseDuration). change interval to a string and parse it on startup. If we don't understand it just do not set up the ticker and warn at that time.